conan/tools/cmake/cmakedeps2/target_configuration.py

```diff
@@ -333,6 +346,12 @@ def _template(self):
             message(STATUS "Conan: Target declared imported {{lib_info["type"]}} library '{{lib}}'")
             add_library({{lib}} {{lib_info["type"]}} IMPORTED)
         endif()
+        {% for alias in lib_info.get("cmake_target_aliases", []) %}
+        if(NOT TARGET {{alias}})
+            message(STATUS "Conan: Target declared alias '{{alias}}' for '{{lib}}'")
+            add_library({{alias}} ALIAS {{lib}})
```

Comment:
This failure comes from older CMakes as per https://cmake.org/cmake/help/latest/command/add_library.html#alias, as it's only from CMake 3.18 onwards that "An ALIAS can target a non-GLOBAL Imported Target"

Reply:
A ``pytest.mark.tool("cmake", "3.27")`` can be added to the necessary tests, no problem with this